tests/unattended/unit/suites/test-elasticsearch.sh

```diff
@@ -37,6 +32,7 @@ test-copyCertificatesElasticsearch() {
 }
 
 test-copyCertificatesElasticsearch-assert() {
+    mkdir /etc/elasticsearch/certs/
```

Comment:
Use `mkdir -p` to make sure that the path exists.

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1188/commits/0d4d12564d2435c1178c68241eed31a20cc52250